contents/testing/integration-testing.md

```diff
@@ -0,0 +1,224 @@
+Description: Enter login credentials and click on login button
+Expected result: To be directed to order food page based on the login user
+~~~
+
+<box type="warning">As integration test cases are <tooltip content="It generally takes longer time due to additional overheads such as waiting for dependent modules to respond">expensive operations</tooltip> compared to unit testing, it should focus mainly on the integration of modules together and not on specific actions within the same module.</box>
+
+:fas-info-circle: More details about the ways to structure incremental testing can be found on [this article](https://www.softwaretestinggenius.com/various-approaches-in-integration-testing/)
+
+### 3. Execute the test scripts and report defects if any
+
+Depending on the approach you have chosen for your integration plan and the test cases, the way you execute your code for testing will differ.
+
+- Big bang approach usually requires all the modules to be developed before you can start with the integration testing.
+
+- Incremental approach is usually conducted simultaneously with the module's development. Stubs and drivers are used to mimic the modules for writing tests since not all modules are developed at the point of integration testing.
+[NUS Software Engineer module site](https://nus-cs2103-ay1819s1.github.io/cs2103-website/se-book-adapted/chapters/testing.html#stubs) provides concrete codes for using stubs.
+
+:fas-exclamation-circle: Either way, such test scripts can help you in singling out specific integration errors to be resolved before it gets more serious. Hence, you should ensure that all high prioritized bugs are fixed before moving on. 
+
+:fas-info-circle: [This article](https://www.guru99.com/test-environment-software-testing.html) shows more details on how to set up a test environment for better integration testing.
+
+### 4. Track defects and re-test application
+
+In the event of failing your integration test case, it is important to learn how to track down the [<tooltip content="Incorrect behavior observed from the system">defects</tooltip>](https://qacomplete.com/resources/articles/what-is-a-software-defect/) occured. 
+Thereafter, you should make changes to your application to fix them and re-test your application with integration testing to ensure that the defects are no longer there.
+
+:fas-info-circle: [This article](http://www.professionalqa.com/defect-tracking-process) covers more details on how to effectively track down defects in a system and fix them.
+
+
+## Projects Adopting Integration Testing
+[Teammates](https://github.com/TEAMMATES/teammates), an online peer evaluations feedback platform, is an exaunit testing, followed by integration testing, then system testing needs to be performed before deploying the product.
+
+Each part of Teammates undergoes unit testing to ensure that each function such as filling up text, selecting dropdowns or checkboxes are working as expected. 
```

Comment:
I think it should be TEAMMATES 😅 
Also, is there a purpose in specifying which projects adopt integration testing? It feels like most projects should be doing it, and identifying certain projects does not necessarily help the reader understand what integration testing is, or why they should use it.